tests/unit/statistical_testing/QuadraticTimeMMD_unittest.cc

```diff
@@ -354,7 +354,7 @@ TEST(QuadraticTimeMMD, perform_test_permutation_biased_full)
 	// assert against local machine computed result
 	mmd->set_statistic_type(ST_BIASED_FULL);
 	float64_t p_value=mmd->compute_p_value(mmd->compute_statistic());
-	EXPECT_NEAR(p_value, 0.0, 1E-10);
+	EXPECT_NEAR(p_value, 0.8, 1E-10);
```

Comment:
@karlnapf the idea was that if someone changes the implementation later and due to some (maybe unintentional) modification, the calculation gives a different result, then this unit test will be able to tell that.